proto/api/v1/file.proto

```diff
@@ -21,3 +21,11 @@ message File {
   string hash = 3;
   int64 size_bytes = 4;
 }
+
+// Request object for DeleteFile
+message DeleteFileRequest {
+  string uri = 1;
```

Comment:
nit: Can you add examples of URIs for:

a CAS artifact
a CAS artifact with a remote_instance_name
an Action Cache artifact
an Action Cache artifact with a remote_instance_name

Reply:
Done

Reply:
I think we want to remove the `blah://cloud.buildbuddy.io/` prefix from these examples because this API is implicitly *for this server*, and we don't want to allow it to try to delete stuff from other, remote, endpoints.

Reply:
Ack